ddl/external_imagerec_prod.hql

```diff
@@ -19,6 +19,8 @@ CREATE EXTERNAL TABLE IF NOT EXISTS `imagerec_prod`(
   `instance_of` string,
   `is_article_page` boolean,
   `dataset_id` string,
+  `insertion_ts` double, 
```

Comment:
This is already added in line 24

Reply:
Urgh. Good catch. 

I'm running a quick integration test to validate the schema maps correctly to parquet.

Reply:
Done! You can validate results at `gmodena.imagerec_prod`.